gsoc/2020/mentors.md

```diff
@@ -17,3 +17,4 @@ layout: plain
 * Jim Pivarski [pivarski@princeton.edu](mailto:pivarski@princeton.edu) Princeton University
 * Alex Richards [a.richards@imperial.ac.uk](mailto:a.richards@imperial.ac.uk) Imperial College London
 * Mark Smith [mark.smith1@imperial.ac.uk](mailto:mark.smith1@imperial.ac.uk) Imperial College London
+* Vassil Vassilev [vvasilev@cern.ch](mailto:vvasilev@cern.ch) Princeton University
```

Comment:
You don't want to add Alexander Penev here as well?

Reply:
Or Bertrand!

Reply:
"and Bertrand" -  all project mentors have to be added here